Packages/com.unity.inputsystem/InputSystem/Plugins/UI/ExtendedPointerEventData.cs

```diff
@@ -106,6 +117,63 @@ internal static int TouchIdFromPointerId(int pointerId)
         /// </summary>
         public Vector2 tilt { get; set; }
         */
+
+        internal void ReadDeviceState()
```

Comment:
Is it worth adding some tests for this method with the different types of device?

Reply:
Not sure, we could maybe add some asserts for `pointerId` and other sensitive information in @Rene-Damm 's `UITests.UI_CanOperateUIToolkitInterface_UsingInputSystemUIInputModule`, make sure that events from the Mouse return `UIElements.PointerId.mousePointerId`, from touchscreen, `UIElements.PointerId.touchPointerIdBase` and +1 if there are two fingers down, and `UIElements.PointerId.penPointerIdBase` for a pen? Rene, do you think it would be simple/useful to do?

Reply:
Yup, think it'd be useful. I'll try to flesh this out in one of the next UI-related PRs. Going to go ahead and merge this one in an effort to try to catch up a bit.